mono/eglib/gunicode-win32-uwp.c

```diff
@@ -19,9 +19,9 @@ g_get_charset (G_CONST_RETURN char **charset)
 {
 	if (my_charset == NULL) {
 		static char buf [14];
-		CPINFOEXA cp_info;
+		CPINFOEXW cp_info;
 
-		GetCPInfoExA (CP_ACP, 0, &cp_info);
+		GetCPInfoExW (CP_ACP, 0, &cp_info);
```

Comment:
Should have one this years ago, whenever Win9x was not supported.

Reply:
I don't think that we have any action to take for this PR, do we?